docs/use/myst.md

```diff
@@ -227,6 +227,71 @@ which automatically numbers the labelled objects.
 For example, `` {numref}`my_figure` `` will produce a reference like:
 {numref}`my_figure`.
 
+### Figure scaling and alignment
+
+Figures can also be aligned by using the option `:align: right` or `:align: left`. By default, figures are aligned to the center (see {numref}`my_figure`).
```

Comment:
Should this be "(see Figure {numref}`my_figure`)" ?

Reply:
nope, by default it includes `Fig.` and in the sphinx configuration there's an option for how you want that formatting to work. (e.g. `Fig` vs.`Figure`)

Reply:
Should we include this information in the documentation as well?

Reply:
I think it's a good idea, since many people will not be familiar with Sphinx

Reply:
Is it possible to set `numfig_format` under `_config.yml`? I tried the option below but got nowhere.
```
numfig_format : {'figure': 'Figure %s', 'table': 'Table %s',
                 'code-block': 'Listing %s', 'section': 'Section %s'}
```

Reply:
Also, do you happen to know the difference between `%s` and `{number}`? I thought at first that `number` was a figure directive option similar to `name`, which can also be referenced as `{name}`.  Now, I'm not so sure.